frameit/lib/frameit/runner.rb

```diff
@@ -1,6 +1,11 @@
 require 'deliver'
```

Comment:
any reason we require the full deliver here?

Reply:
Good question, I probably did `require 'deliver'` and wanted to replace it with a more specific version later, then forgot

Reply:
In case that helps, here are occurences of similar large requires:

`Dir.glob("*/lib").map{|s| s.split('/')[0]}.each {|tool| command = "git grep \"require '#{tool}'\""; puts `#{command}`}`

(to run in a irb shell at root)

Reply:
Upon further checking, deliver is not actually used in this file at all 😅 I assume the file was probably just copied way back when frameit was made, or something, and that was left over. I'm sure there are some unnecessary requires over all of fastlane. 

I think my MissingRequireStatement cop could even warn for those, since `require` returns `false` if the file was already loaded... hmm...